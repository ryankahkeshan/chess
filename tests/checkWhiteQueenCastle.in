setup 
+ K e1 + k e8
+ R a1 + r a8
+ N f3 + N c4 + N c5 + n b3
+ b f5 + b g5
= black
done game human human
move g5 h4

move e1 c1
move f3 h4
move e8 c8

move e1 c1
move c4 d6

move c8 b7
move c8 c7

move e1 c1
move c5 b3

move c7 b7
move c7 d6
move e1 c1
