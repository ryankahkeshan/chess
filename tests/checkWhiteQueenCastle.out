8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _N_ _ _
3 _ _ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _ _ 
4  _N_ _ _
3 _ _ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _ _ 
4  _N_ _ _
3 _n_ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _b_ 
4  _N_ _ _
3 _n_ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _bb 
4  _N_ _ _
3 _n_ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _bb 
4  _N_ _ _
3 _n_ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _b_ 
4  _N_ _ b
3 _n_ _N_ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

White is in check.
Invalid move
8 r_ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _b_ 
4  _N_ _ N
3 _n_ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8  _kr _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ N _b_ 
4  _N_ _ N
3 _n_ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

Invalid move
8  _kr _ _
7 _ _ _ _ 
6  _ N _ _
5 _ N _b_ 
4  _ _ _ N
3 _n_ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

Black is in check.
Invalid move
8  _ r _ _
7 _ k _ _ 
6  _ N _ _
5 _ N _b_ 
4  _ _ _ N
3 _n_ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

Invalid move
8  _ r _ _
7 _ k _ _ 
6  _ N _ _
5 _ _ _b_ 
4  _ _ _ N
3 _N_ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

Invalid move
8  _ r _ _
7 _ _ _ _ 
6  _ k _ _
5 _ _ _b_ 
4  _ _ _ N
3 _N_ _ _ 
2  _ _ _ _
1 R _ K _ 

  abcdefgh

8  _ r _ _
7 _ _ _ _ 
6  _ k _ _
5 _ _ _b_ 
4  _ _ _ N
3 _N_ _ _ 
2  _ _ _ _
1 _ KR_ _ 

  abcdefgh

Black is in check.
Final Score:
White: 0
Black: 0
